### ISSUE TITLE
handle not working some times

### ISSUE THREAD

**Issue by M-Barari** [rlemaigre/Easy-DnD#77]
Hi. I have a weird issue! I add handle to drag and it works if the handle class in in the same component as drag:
```vue
<template>
    <div class="pa-4">
        <drop-list :items="items" @reorder="mm" class="row ma-4">
            <template v-slot:item="{item,reorder}">
                <drag :key="item.title" class="col col-3" :data="item" handle=".handle">
                    <v-chip class="handle" :color="reorder ? 'primary' : null">{{item.title}}</v-chip>
                </drag>
            </template>
        </drop-list>
    </div>
</template>
```

but if the handle class is in a different component, on some components it works and on some don't!!
```vue
<template>
    <div class="pa-4">
        <drop-list :items="items" @reorder="mm" class="row ma-4">
            <template v-slot:item="{item,reorder}">
                <drag :key="item.title" class="col col-3" :data="item" handle=".handle">
                    <!-- the 'handle' class is in component -->
                    <my-component />
                </drag>
            </template>
        </drop-list>
    </div>
</template>
```
my component:
```vue
<template>
    <v-card flat class="cms_product_list_card">
        <!-- HERE IS THE HANDLE -->
        <div class="handle sorting_wrapper pa-1 py-2 d-flex flex-column align-center justify-space-between" height="100%">
            <div><v-icon class="text_color_white" medium>mdi-sort-variant</v-icon></div>
            <div class="productlist__btn__sort">{{lang.holdtoorder}}</div>
            <v-avatar class="count" color="white" size="24"><span class="text_side_color">{{index+1}}</span></v-avatar>
        </div>
        <div class="info_wrapper d-flex flex-column justify-space-between">
            <div class="full_width d-flex mb-1 pa-1">
                <v-img class="product_img" contain :alt="product.title" :src="imageUrl" />
                <div class="full_width ps-2 pe-1 d-flex flex-column justify-space-between">
                    <p class="product_title">
                        {{product.title}}
                        <pillfeatured v-if="isTimedFeatured" :time="product.featured" />
                    </p>
                    <div class="d-flex justify-space-between">
                        <div>
                            <pills :options="{isFeatured: isFeatured,isPopular: isPopular,isNew: isNew}" />
                            <div class="d-flex align-center mt-2">
                                <v-rating 
                                half-icon="mdi-star-half-full"
                                :value="starNum" 
                                readonly 
                                background-color="yellow accent-4" 
                                color="yellow accent-4"  
                                half-increments
                                ></v-rating>
                                <p class="mb-0 rate_customer">{{customerNum}}</p>
                            </div>
                        </div>
                        <div class="d-flex flex-column align-end justify-end">
                            <template v-if="isOff">
                                <div class="d-flex align-center">
                                    <v-avatar class="off_wrapper me-1">
                                        <span class="off_percent">{{offPercent}}%</span>
                                    </v-avatar>
                                    <div class="off_price">
                                        {{formattedPrice(product.price.firstPrice)}}
                                    </div>
                                </div>
                            </template>
                            <template>
                                <div class="main_price">
                                    {{formattedPrice(product.price.finalPrice)}}
                                </div>
                            </template>
                        </div>
                    </div>
                </div>
            </div>
    </div>
</template>
```
 can anybody help me??

**Comment by bensoutendijk** [comment]
I am having the same issue. Also, even when you do include the `.handle` element directly in the prop, it does not always work if there is a `v-if` directive.

My solution is to wrap `<MyComponent />` in a div, and make sure that the `.handle` element is a direct child of that div. Then I used `v-show` instead of `v-if`

**Comment by rlemaigre** [comment]
Fixed in 1.12.1

**Comment by M-Barari** [comment]
That problem is fixed but it looks like this fix messed up with mobile touch !! It won't drag on touch devices anymore
